src/vs/workbench/contrib/scm/browser/activity.ts

```diff
@@ -224,8 +224,9 @@ export class SCMActiveRepositoryContextKeyController implements IWorkbenchContri
 	private activeRepositoryNameContextKey: IContextKey<string>;
 	private activeRepositoryBranchNameContextKey: IContextKey<string>;
 
+	private focusedRepository: ISCMRepository | undefined = undefined;
+	private focusDisposable: IDisposable = Disposable.None;
```

Comment:
My preferred pattern here is `MutableDisposable` which allows you to register possible disposables at the time of object creation, so you wouldn't need to implement/override a `dispose` function.